runtime/memprof.c

```diff
@@ -526,4 +525,6 @@ void caml_memprof_track_interned(header_t* block, header_t* blockend) {
 
     p = next_p;
   }
+
+  caml_memprof_handle_postponed();
```

Comment:
If we do this, then I don't see the point of not calling the other callbacks. And if call the other callbacks, I don't see why we should not do this in `intern.c` as before.